pkgs/applications/editors/vscode/extensions/eamodio.gitlens/default.nix

```diff
@@ -0,0 +1,105 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  pnpm,
+  nodejs,
+  vscode-utils,
+  nix-update-script,
+}:
+
+let
+  vscode-src = fetchFromGitHub {
```

Comment:
Why not use vscode.src?

Reply:
Because vscode package is unfree and vscode.src is only a zip package without source code.